### PR TITLE
Management: Project Manager, Program Manager, and Portfolio Manager

### PR DESCRIPTION
**Pull request by nhuthm**
- Difference between manager roles

### CODE REVIEW

**Comment by nhuthm** [comment]
@monotykamary please help to take a review on this one mate

File: Engineering/Management/Difference betwwen Project-Program-Portfolio manager.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+For this person, we can see that he/she will be the one who is incharge of planning, scheduling, budgeting, execution, and delivery of the product. Basically, they are the one who will run the team based on their leadership skills, technical knowledge, and experiences to direct their members. Also, the most important is, the software is meet with the client's requirements.
+
+##Program Manager
+In the Product team, we might have quite a lot of team to work on the different features that are working on the big system. Right now, the Program Manager is the one who is incharge of the work for the team project. Basically, for this role, it's just higher than Project Manager. If the Project Manager is the one who want to make sure the project's goal reach client's requirement, the Program Manager is the one who will decide which team will work on the requirement.
+
+For example: if your product are having about 5 or 6 teams, the Program Manager is assigned to manage the team number 2nd and 3rd. When receiving the reqest, the Program Manager will think about which team that is he/she is incharged, to work on this request so the team can deliver successfully.
```
Comment (janrvu) [inline Engineering/Management/Difference betwwen Project-Program-Portfolio manager.md:16]: spelling mistake: When receiving the reqest --> When receiving the request

File: Engineering/Management/Difference betwwen Project-Program-Portfolio manager.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Portfolio Manager
+Can say, this role is the most important than others 2 mention roles above. The reason is: they will be the one who have to prioritize the work for every teams, every programs that they are managing. After that, they will know which program should do the stuffs before providing informations to project team. Basically, can see that they are mostly focusing on the businees site to balance the workloads.
```
Comment (janrvu) [inline Engineering/Management/Difference betwwen Project-Program-Portfolio manager.md:19]: spelling mistake: focusing on the businees site  --> business

File: Engineering/Management/Difference betwwen Project-Program-Portfolio manager.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## The difference between those roles
+Basically, for those 3 roles: Project Manager, Program Manager, and Portfolio Manager are in common of 1 term: Management. The big difference between them is: the resposible on each role that is playing in the Product team to make everything is working correctly and the output is reach with the client's requirements.
```
Comment (janrvu) [inline Engineering/Management/Difference betwwen Project-Program-Portfolio manager.md:22]: spelling mistake: the resposible on each role --> the responsible for each role

**Review by janrvu — COMMENTED** [review]
The note is presented in the correct format. There is some spelling mistake that needs to edit.

**Comment by nhuthm** [comment]
@janrvu please help me to have another review, had updated the wrong typo

**Review by janrvu — COMMENTED** [review]
All typos have been corrected. Thanks Nhut for the update.

**Review by monotykamary — APPROVED** [review]
LGTM

**Pull request merged**